pkg/consts/otel.go

```diff
@@ -0,0 +1,48 @@
+package consts
+
+const (
+	// system attributes
+	OtelSysAccountID      = "sys.account.id"
+	OtelSysWorkspaceID    = "sys.workspace.id"
+	OtelSysAppID          = "sys.app.id"
+	OtelSysEventData      = "sys.event"
+	OtelSysEventRequestID = "sys.event.request.id"
+	OtelSysEventIDs       = "sys.event.ids"
+	OtelSysBatchID        = "sys.batch.id"
+	OtelSysIdempotencyKey = "sys.idempotency.key"
+
+	OtelSysFunctionID         = "sys.function.id"
+	OtelSysFunctionSlug       = "sys.function.slug"
+	OtelSysFunctionVersion    = "sys.function.version"
+	OtelSysFunctionScheduleAt = "sys.function.time.schedule"
+	OtelSysFunctionStartAt    = "sys.function.time.start"
+	OtelSysFunctionEndAt      = "sys.function.time.end"
+	OtelSysFunctionStatus     = "sys.function.status"
+	OtelSysFunctionOutput     = "sys.function.output"
+
+	OtelSysStepScheduleAt      = "sys.step.time.schedule"
+	OtelSysStepStartAt         = "sys.step.time.start"
+	OtelSysStepEndAt           = "sys.step.time.end"
+	OtelSysStepStatus          = "sys.step.status"
+	OtelSysStepAttempt         = "sys.step.attempt"
+	OtelSysStepOutput          = "sys.step.output"
+	OtelSysStepOutputSizeBytes = "sys.step.output.size.bytes"
+
+	// SDK attributes
+	OtelAttrSDKServiceName = "sdk.app.name"
+	OtelAttrSDKRunID       = "sdk.run.id"
```

Comment:
Main reason for not making this `sys` prefix is because we'll expect to ship these data to customers when we export the data to their tooling.
The `sys` ones are the data we should filter out since those information is likely useless for them.